requirements/docker.txt

```diff
@@ -1,5 +1,5 @@
 --no-index
+--find-links https://pyrepo.stage.mozaws.net/wheelhouse/
 --find-links https://pyrepo.stage.mozaws.net/olympia/
---find-links https://pyrepo.stage.mozaws.net/wheelhouse
 
```

Comment:
Please don't ask but for some reasons m2crypto fails if not installing from `wheelhouse` instead of `olympia`. Afair we're using `olympia` in production and `wheelhouse` is near enough for local development to represent our production environment. If m2crypto is liked against another minor version of libssl I think that's okay.